src/Dev/TestSession.php

```diff
@@ -235,7 +236,7 @@ public function submitForm(string $formID, string $button = null, array $data =
             if ($button) {
                 $btnXpath = "//button[@name='$button'] | //input[@name='$button'][@type='button' or @type='submit']";
                 if (!$formCrawler->children()->filterXPath($btnXpath)->count()) {
-                    throw new Exception("Can't find button '$button' to submit as part of test.");
+                    throw new LogicException("Can't find button '$button' to submit as part of test.");
```

Comment:
Not really necessary but it's a better exception than just the catch-all `Exception`